packages/near-membrane-base/src/environment.ts

```diff
@@ -38,31 +37,26 @@ interface VirtualEnvironmentOptions {
     blueGlobalThis: BlueObject & typeof globalThis;
     // Red global object used by the red environment
     redGlobalThis: RedObject & typeof globalThis;
-    // Optional distortion map to tame functionalities observed through the membrane
-    distortionMap?: Map<RedProxyTarget, RedProxyTarget>;
+    // Optional distortion callback to tame functionalities observed through the membrane
+    distortionCallback?: (originalTarget: RedProxyTarget) => RedProxyTarget;
 }
 
+const distortionNoopCallback = (v: RedProxyTarget) => v;
```

Comment:
☝️ This isn't technically a noop. This is an [identity function](https://en.wikipedia.org/wiki/Identity_function). Like PrototypeJS's `Prototype.K` function or Lodash's `_.identity`.

Reply:
will rename it.